backstop_data/engine_scripts/puppet/clickAndHoverHelper.js

```diff
@@ -0,0 +1,39 @@
+module.exports = async (page, scenario) => {
```

Comment:
We're using playwright, yeah? So everything in `backstop_data/engine_scripts/puppet` can be removed.